### PR TITLE
chore: some linter based cleanup

### PR DESCRIPTION
**Pull request by henryiii**
Tried running `hatch fmt`, and applied most of the fixes. (I don't agree with all of it's default checks, but it's customizable and I could set it up in the future). Then manually ran `ruff check .` with some different `--select` rules to find a few more things for easy cleanup. Not going all out adding https://learn.scientific-python.org/development/guides/style/ and enforcing anything but this looked like easy cleanup.

- **style: some touch up from hatch fmt**
- **chore: some more manual fixes from running ruff**

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -31,10 +27,10 @@
     "use_issues_button": True,
     "use_edit_page_button": True,
     "show_toc_level": 3,
-    'default_mode': 'dark',  
+    'default_mode': 'dark',
 }
 html_logo = "./logo.svg"
 html_favicon = "./logo.svg"
-html_title = '{0}'.format(project)
+html_title = f'{project}'
```
Comment (henryiii) [inline docs/conf.py:34]:
```suggestion
html_title = project
```

File: peytonites/core.py [hunk 2]
```diff
@@ -146,7 +145,7 @@ def plot(self, unit='cm'):
         coordinates = [(x, y), (x, z), (y, z)]
         labels = [('X', 'Y'), ('X', 'Z'), ('Y', 'Z')]
 
-        for ax, (data_x, data_y), (label_x, label_y) in zip(axs, coordinates, labels):
+        for ax, (data_x, data_y), (label_x, label_y) in zip(axs, coordinates, labels, strict=False):
```
Comment (henryiii) [inline peytonites/core.py:148]: This maybe should be strict=True, but that's a change in behavior.

**Review by robelgeda — APPROVED** [review]
Interesting, I was told to do `"{}".format` for python3.x compat reasons but I realize this is not longer true. lgtm

**Comment by henryiii** [comment]
`"{}".format` is for Python 2 and 3.5 compat. `"{0}".format` is for Python 2.6 compat. :)

**Pull request merged**